tests/unit/wer_report_test_wrapper.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define _wer_report _wer_report_test
+#define AddVectoredExceptionHandler AddVectoredExceptionHandler_test
+#define SetThreadStackGuarantee SetThreadStackGuarantee_test
+#define RemoveVectoredExceptionHandler RemoveVectoredExceptionHandler_test
+#define WerReportCreate WerReportCreate_test
+#define WerReportAddDump WerReportAddDump_test
+#define WerReportSubmit WerReportSubmit_test
+#define WINBASEAPI
+
+// Included first to ensure the overridden copy is the only one.
+#include "wer_report.hpp"
+
+#undef _wer_report
+#undef AddVectoredExceptionHandler
+#undef SetThreadStackGuarantee
+#undef RemoveVectoredExceptionHandler
+#undef WerReportCreate
+#undef WerReportAddDump
+#undef WerReportSubmit
+#undef WINBASEAPI
+#undef _dupenv_s
+
+#include "catch_wrapper.hpp"
+
+uint32_t _vectored_exception_handler_test = 1;
+
+ULONG AddVectoredExceptionHandler_test_first = 0;
+PVECTORED_EXCEPTION_HANDLER AddVectoredExceptionHandler_test_handler = nullptr;
+
+_Ret_maybenull_ PVOID
+AddVectoredExceptionHandler_test(_In_ ULONG first, _In_ PVECTORED_EXCEPTION_HANDLER handler)
```

Comment:
```suggestion
AddVectoredExceptionHandler_test(ULONG first, _In_ PVECTORED_EXCEPTION_HANDLER handler)
```
Per SAL documentation, `_In_` is only for pointers, not value types.

Reply:
SAL needs to match the wrong annotation in the Windows header.